README.md

```diff
@@ -137,15 +135,17 @@ The configuration is done by [Typesafe config](https://github.com/typesafehub/co
 
 ```
 druid = {
-  host = "localhost"
-  host = ${?DRUID_HOST}
-  port = 8082
-  port = ${?DRUID_PORT}
+  hosts = "localhost:8082"
+  hosts = ${?DRUID_HOSTS}
   secure = false
   secure = ${?DRUID_USE_SECURE_CONNECTION}
   url = "/druid/v2/"
   url = ${?DRUID_URL}
-
+  health-endpoint = "/status/health"
+  health-endpoint = ${?DRUID_HEALTH_ENDPOINT}
+  client-backend = "ing.wbaa.druid.client.DruidHttpClient"
+  client-backend = ${?DRUID_CLIENT_BACKEND
```

Comment:
There is a `}` missing here

Reply:
fixed :)

Reply:
I wasnn't aware I already sent this comment :) will continue the review now